test/init.test.js

```diff
@@ -95,6 +96,6 @@ describe('init', () => {
 
     test('invalid template name passed', async () => {
         const { stdout, stderr } = await exec(`node ../../bin/index.js init -t invalid_name`, { cwd: tempDir });
-        expect(stderr).toContain(`Template invalid_name does not exist. To see available templates use "qse list".`);
+        expect(stripAnsi(stderr)).toContain(`Template invalid_name does not exist. To see available templates use "qse list".`);
     });
```

Comment:
Add more test cases for invalid names and also valid name.

### Invalid cases I can think of

- `>214` chars.
- uppercase.
- include space in name.

### Valid cases I can think of

- `<= 214` chars
- lowercase-only
- any other valid name.

Reply:
I have added the test cases.